src/OrchardCore.Modules/OrchardCore.Email.Azure/Manifest.cs

```diff
@@ -1,14 +1,14 @@
 using OrchardCore.Modules.Manifest;
 
 [assembly: Module(
-    Name = "Azure Email Provider",
+    Name = "Azure Communication Email",
     Author = ManifestConstants.OrchardCoreTeam,
     Website = ManifestConstants.OrchardCoreWebsite,
     Version = ManifestConstants.OrchardCoreVersion,
-    Description = "Provides an email service provider leveraging Azure Communication Services (ACS).",
+    Description = "Provides email service providers leveraging Azure Communication Services (ACS).",
     Dependencies =
     [
-        "OrchardCore.Email"
+        "OrchardCore.Email",
```

Comment:
Still wondering while you like the comma

Reply:
we do this everywhere. we talked about it so many times. When you have a comma, then any time we have array insertion, you'll see one line change instead of two. Sorry I don't want to debt this again